playbooks/deploy_edpm_os_configure.yml

```diff
@@ -64,9 +64,21 @@
         tasks_from: configure.yml
       tags:
         - edpm_timezone
+    - name: configure edpm_frr
+      ansible.builtin.import_role:
+        name: osp.edpm.edpm_frr
+        tasks_from: configure.yaml
+      tags:
+       - edpm_frr
     - name: Configure edpm_ovn
       ansible.builtin.import_role:
         name: osp.edpm.edpm_ovn
         tasks_from: configure.yml
       tags:
         - edpm_ovn
+    - name: configure edpm_ovn_bgp_agent
```

Comment:
Task name should be start with an uppercase letter (see https://github.com/ansible/ansible-lint/blob/main/src/ansiblelint/rules/name.py#L127)